pkgs/games/tome2/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub,  makeDesktopItem, ncurses, libX11, boost, cmake }:
+
+let
+  pname = "tome2";
+  description = "A dungeon crawler similar to Angband, based on the works of Tolkien.";
+
+  desktop = makeDesktopItem {
```

Comment:
You don't reference this anywhere.

Reply:
Fixed — I hope that's the right way to do it.